File: domain/src/main/java/no/nav/melosys/domain/person/Informasjonsbehov.java
package no.nav.melosys.domain.person;

public enum Informasjonsbehov {
    INGEN("Uten tilleggsopplysninger"),
    STANDARD("Med adresseopplysninger"),
    MED_FAMILIERELASJONER("Med adresseopplysninger og familierelasjoner");

    private String beskrivelse;

    Informasjonsbehov(String beskrivelse) {
        this.beskrivelse = beskrivelse;
    }

    public String getBeskrivelse() {
        return beskrivelse;
    }
}

File: domain/src/main/kotlin/no/nav/melosys/domain/person/Informasjonsbehov.kt
package no.nav.melosys.domain.person

enum class Informasjonsbehov(private val beskrivelse: String) {
    INGEN("Uten tilleggsopplysninger"),
    STANDARD("Med adresseopplysninger"),
    MED_FAMILIERELASJONER("Med adresseopplysninger og familierelasjoner");
}
